widget/playground/src/constants/languages.ts

```diff
@@ -1,12 +1,8 @@
+import { English, French, Japanese, Spanish } from '@rango-dev/ui';
```

Comment:
When I reset the config, the language doesn't change. It's becuae of `if (config?.language)` in `Widget.tsx`.
It can be changed to `  useEffect(() => {
      changeLanguage(config?.language || defaultLanguage);
  }, [config?.language]);` or another solution that you'll find.

Reply:
your change is ok btw useLanguage already exports `defaultLanguage` you can get from it.